ring_doorbell/stickup_cam.py

```diff
@@ -27,17 +27,17 @@ def model(self):
         """Return Ring device model name."""
         if self.kind in FLOODLIGHT_CAM_KINDS:
             return 'Floodlight Cam'
-        elif self.kind in SPOTLIGHT_CAM_BATTERY_KINDS:
+        if self.kind in SPOTLIGHT_CAM_BATTERY_KINDS:
```

Comment:
Same for this sequence here...

Reply:
@tchellomello  I've reverted these sections and added a comment that the linter error (when running tox -r) for no-else-return can be ignored